cmd/antrea-controller/controller.go

```diff
@@ -111,10 +111,7 @@ func run(o *Options) error {
 	stopCh := signals.RegisterSignalHandlers()
 
 	informerFactory.Start(stopCh)
-	// Only start watching Security CRDs when config option is set to true.
-	if o.config.EnableSecurityCRDs {
-		crdInformerFactory.Start(stopCh)
-	}
+	crdInformerFactory.Start(stopCh)
```

Comment:
As an experimental feature, I think we should still disable it by default and enable it by option, perhaps based on https://github.com/vmware-tanzu/antrea/pull/847

Reply:
Will rebase this patch onto #847 as soon as it is merged

Reply:
Rebased.